evals/elsuite/audio/eval.py

```diff
@@ -60,8 +62,18 @@ def eval_sample(self, sample: Sample, rng):
         sampled = self.do_completion(prompt, **kwargs)
         return self.compute_metrics(sample, sampled)
 
+    def _keep_sample(self, sample, max_audio_duration: int):
+        """
```

Comment:
Suggest omitting duration as an argument, you can get it from self.

Reply:
Okay sure. I was going back & fourth too, the reason I kept it was to make it explicit that this should be handled. I'll switch then.